docs/src/styleguide.md

```diff
@@ -55,7 +55,7 @@ julia -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.6
 ```
 You can then recursively format the core Julia files in the Trixi.jl repo by executing
 ```shell
-julia -e 'using JuliaFormatter; format(["benchmark", "examples", "ext", "src", "test", "utils"])'
+julia -e 'using JuliaFormatter; format(".")'
```

Comment:
```suggestion
julia -e 'using JuliaFormatter; format(".")'
```
Ah, now you see that you already adjusted that 👍. We could also simply use `"."` now instead of listing all folders separately.

Reply:
No worries it's Ok